js/enter_game.js

```diff
@@ -1,81 +1,144 @@
-// helper functions
-function hideElment (id, hide) {
-  const element = document.getElementById(id) // textinput A
-  if (hide) {
-    element.style.display = 'none'
-  } else {
-    element.style.display = 'inline'
-  }
-}
+/* global $ apiPost */
+var tok2 = null;
```

Comment:
I guess code is formatted correctly by now? Standard says '$' is not defined at the tokenize2 configuration of enter_game, but this cannot be changed, as it does not work otherwise.

Reply:
See  6af7375